### ISSUE TITLE
Store and write the project version

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/gdscript-docs-maker#36]
We should get the project version when generating the code reference, so users know which version of the tool they're reading docs for. Also, this could allow more complex projects to keep several major versions of their tools' docs in parallel, like Godot has 2.x, 3.0, 3.1, stable, and latest branches. I.e. they could output several files based on the project's Godot version.

It should be extracted from project.godot, I believe there's a version string in there.

**Comment by Razoric480** [comment]
It's actually stored in the `export_presets.cfg`, and is set per-preset. Under GDScript it'd be something like:
```gdscript
var file := File.new()
if file_exists("res://config_preset.cfg"):
	var export_config := ConfigFile.new()
	var config_error := export_config.load("res://config_preset.cfg")
	if config_error == OK:
		var product_version: String = get_value("preset.0.options", "application/product_version")
		if product_version:
			print(product_version)
```

**Comment by Razoric480** [comment]
There's also [this plugin](https://github.com/FuzzyRhombus/Godot-Version) that I stumbled on, though it's for 2.1 and it hasn't been updated in 2 years.

**Comment by Calinou** [comment]
[This pull request](https://github.com/godotengine/godot/pull/35555) should solve this issue once it's merged :slightly_smiling_face:

**Comment by NathanLovato** [comment]
Thanks, I'll look at the version format and implement something sometime

**Comment by NathanLovato** [comment]
How about writing this? Where should I put the version number? The tool generates standalone markdown files for each class. Should we write the version number in each of them? Or should we write an index.md file with e.g. a TOC and the main project information?

**Comment by Razoric480** [comment]
The project.godot DOES have a Project Name and Project Description. Seems like pretty apt to put that in a main index.

And if we add categories, we can make the TOC work pretty well. Which brings us back to #29

**Comment by NathanLovato** [comment]
Alright I'll mark this as solved and move the requirements to #29
